### PR TITLE
 Adds the golem Time-Green was too communistic to add

### PR DESCRIPTION
**Pull request by fluffe9911**
[Patriotic Capitalist Golem Demonstration](https://www.youtube.com/watch?v=C0wrCTjyM-o&feature=youtu.be) (imagine the song is using the U.S Anthem instead)

Capitalisium much like stalinium is currently unavailable, so it's also a admin only golem also could see some good events with this like capitalist golems vs soviet golems eh? 

🆑 Fluffe9911
rscadd: Adds industrialist capitalist golem!
/🆑

### CODE REVIEW

**Comment by fluffe9911** [comment]
will fix merge conflicts in a moment

**Comment by fluffe9911** [comment]
just fixed it as you posted that lol

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 1]
```diff
@@ -1011,6 +1011,46 @@
...
 
+/datum/species/golem/capitalist
+	name = "Capitalist Golem"
+	id = "capitalist golem"
+	prefix = "Capitalist"
+	attack_verb = "monopoliz"
+	limbs_id = "ca_golem"
+	blacklisted = 1
+	special_names = list("John D. Rockefeller","Rich Uncle Pennybags","Commodore Vanderbilt","Entrepreneur","Mr Moneybags", "Adam Smith")
+	species_traits = list(NOBLOOD,NO_UNDERWEAR,NOEYES)
```
[the remark this answers is not in the document]
Comment (fluffe9911) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: theirs actually a small difference with this one so this part kinda has to stay 
(plus the sprite just feken breaks if you remove it)

**Comment by PKPenguin321** [comment]
image
**🇺🇸🇺🇸🇺🇸🇺🇸 GOD BLESS, BROTHER 🇺🇸🇺🇸🇺🇸🇺🇸**
image

**Comment by Anonmare** [comment]
AMERICA
FUCK YEAH

**Comment by Jordie0608** [comment]
This is cool and all but those two sounds are copyrighted.

**Comment by 81Denton** [comment]
You know what would be good
The bananium golem thing where they bleed bananas except this guy bleeds cash and hamburgers

**Comment by fluffe9911** [comment]
@Jordie0608 I mean one is guile them from a old snes game and the other is a literal 2 second clip from the spongebob movie doubt anyones going to raise any problems with it

**Comment by Ziiro** [comment]
why don't they get guns

these are hardly american

**Comment by fluffe9911** [comment]
They are more a representation of capitalism in general then america really

**Comment by fluffe9911** [comment]
Capitalist golems now bleed money lol

**Comment by drderp3635** [comment]
Noguns trait, litterally unplayable

**Comment by fluffe9911** [comment]
Some things must be sacrificed in the name of balance

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 2]
```diff
@@ -1021,7 +1092,6 @@
 	special_names = list("Stalin","Lenin","Trotsky","Marx","Comrade") //comrade comrade
 	species_traits = list(NOBLOOD,NO_UNDERWEAR,NOEYES)
 	fixed_mut_color = null
-	inherent_traits = list(TRAIT_NOBREATH, TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOGUNS,TRAIT_RADIMMUNE,TRAIT_PIERCEIMMUNE,TRAIT_NODISMEMBER)
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: what's this about?

Comment (ATH1909) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: Am I misinterpreting things, or did you just reset soviet golems to having the default golem traits?

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 3]
```diff
@@ -1011,6 +1011,77 @@
...
+	if(isgolem(target))
+		return
+	if(target.nutrition >= NUTRITION_LEVEL_FAT)
+		target.set_species(/datum/species/golem/capitalist)
+		return
+	target.adjust_nutrition(40)
+
+/datum/species/golem/capitalist/handle_speech(message, mob/living/carbon/human/H)
+	playsound(H, 'sound/misc/mymoney.ogg', 25, 0)
+	return "Hello I like money!"
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: Shouldn't there be a comma after "Hello"?

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 4]
```diff
@@ -1011,6 +1011,77 @@
...
+	C.equip_to_slot_or_del(new /obj/item/clothing/head/that (), SLOT_HEAD)
+	C.equip_to_slot_or_del(new /obj/item/clothing/glasses/monocle (), SLOT_GLASSES)
+	C.revive(full_heal = 1)
+
+	SEND_SOUND(C, sound('sound/misc/capitialism.ogg'))
+	C.AddSpell(new /obj/effect/proc_holder/spell/aoe_turf/knock ())
+
+/datum/species/golem/capitalist/on_species_loss(mob/living/carbon/C)
+	. = ..()
+	for(var/obj/effect/proc_holder/spell/aoe_turf/knock/spell in C.mob_spell_list)
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: I feel like there has to be a better spell for capitalist golems than knock.

Maybe give them that devil boon spell that summons money?

If you go the route of removing their TRAIT_NOGUNS trait, you HAVE to make their racial spell lesser summon guns (the one that summons a single-shot rifle into your hands).

File: code/modules/mob/living/carbon/human/species_types/golems.dm (hunk 2 above)
Comment (fluffe9911) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: yeah but there was no differences except for resist heat being removed for some reason so it was redundant

File: code/modules/mob/living/carbon/human/species_types/golems.dm (hunk 3 above)
Comment (fluffe9911) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: I guessss

File: code/modules/mob/living/carbon/human/species_types/golems.dm (hunk 4 above)
Comment (fluffe9911) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: I mean heres the problem I kinda want this to be balanced around capitalist golems vs soviet golems so if I remove the NO_GUNS I would do it for the soviet golems as well and vice versa gotta keep it balanced

File: code/modules/mob/living/carbon/human/species_types/golems.dm (hunk 2 above)
Comment (fluffe9911) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: although may readd it once I remove the NO_GUNS THING

**Comment by drderp3635** [comment]
>balance concern on an admeme only race that converts the entire station to itself

**Comment by fluffe9911** [comment]
balance against eachother not in general lol

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 5]
```diff
@@ -1011,6 +1011,78 @@
...
+/datum/species/golem/capitalist/spec_attacked_by(obj/item/I, mob/living/user, obj/item/bodypart/affecting, intent, mob/living/carbon/human/H)
+	..()
+	if(world.time > last_cash + cash_cooldown && user != H)
+		new/obj/item/stack/spacecash/c20(get_turf(H))
+		last_cash = world.time
+
+/datum/species/golem/capitalist/on_hit(obj/item/projectile/P, mob/living/carbon/human/H)
+	..()
+	if(world.time > last_cash + cash_cooldown)
+		new/obj/item/stack/spacecash/c20(get_turf(H))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]:
```suggestion
		new/obj/item/stack/spacecash/c20(H.drop_location())
```

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 6]
```diff
@@ -1011,6 +1011,78 @@
...
+	..()
+	if(world.time > last_cash + cash_cooldown && M != H &&  M.a_intent != INTENT_HELP)
+		new/obj/item/stack/spacecash/c20(get_turf(H))
+		last_cash = world.time
+
+/datum/species/golem/capitalist/spec_attacked_by(obj/item/I, mob/living/user, obj/item/bodypart/affecting, intent, mob/living/carbon/human/H)
+	..()
+	if(world.time > last_cash + cash_cooldown && user != H)
+		new/obj/item/stack/spacecash/c20(get_turf(H))
+		last_cash = world.time
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: Duplicated code, should be refactored into a separate proc

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 7]
```diff
@@ -1011,6 +1011,78 @@
...
+/datum/species/golem/capitalist/spec_hitby(atom/movable/AM, mob/living/carbon/human/H)
+	..()
+	var/obj/item/I
+	if(istype(AM, /obj/item))
+		I = AM
+		if(I.thrownby == H) //No throwing stuff at yourself to make cash
+			return 0
+		else
+			new/obj/item/stack/spacecash/c20(get_turf(H))
+			last_cash = world.time
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: Same

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 8]
```diff
@@ -1011,6 +1011,78 @@
...
+		new/obj/item/stack/spacecash/c20(get_turf(H))
+		last_cash = world.time
+
+/datum/species/golem/capitalist/spec_hitby(atom/movable/AM, mob/living/carbon/human/H)
+	..()
+	var/obj/item/I
+	if(istype(AM, /obj/item))
+		I = AM
+		if(I.thrownby == H) //No throwing stuff at yourself to make cash
+			return 0
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: Is the return value for this proc used?

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 9]
```diff
@@ -1011,6 +1011,78 @@
...
+
+/datum/species/golem/capitalist/handle_speech(message, mob/living/carbon/human/H)
+	playsound(H, 'sound/misc/mymoney.ogg', 25, 0)
+	return "Hello I like money!"
+
+/datum/species/golem/capitalist/spec_attack_hand(mob/living/carbon/human/M, mob/living/carbon/human/H, datum/martial_art/attacker_style)
+	..()
+	if(world.time > last_cash + cash_cooldown && M != H &&  M.a_intent != INTENT_HELP)
+		new/obj/item/stack/spacecash/c20(get_turf(H))
+		last_cash = world.time
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: More dupe

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 10]
```diff
@@ -1011,6 +1011,78 @@
...
+	info_text = "As a <span class='danger'>Capitalist Golem</span>, your fist spreads the powerful industrializing light of capitalism."
+
+	var/last_cash = 0
+	var/cash_cooldown = 100
+
+/datum/species/golem/capitalist/on_species_gain(mob/living/carbon/C, datum/species/old_species)
+	. = ..()
+	C.equip_to_slot_or_del(new /obj/item/clothing/head/that (), SLOT_HEAD)
+	C.equip_to_slot_or_del(new /obj/item/clothing/glasses/monocle (), SLOT_GLASSES)
+	C.revive(full_heal = 1)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]:
```suggestion
	C.revive(full_heal = TRUE)
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by AnonymousNow** [comment]
Doesn't make sense, newly-made capitalist golems need to start with less equipment than the golem that makes them.

**Comment by Jordie0608** [comment]
@fluffe9911 copyright is more about the principle of not adding more copyrighted content without correct licensing or attribution than worrying if we'd actually be sued or not.

**Comment by fluffe9911** [comment]
@Jordie0608 feel free to give some suggestions for some non copyrighted alternatives that sound good then lol

**Comment by ExcessiveUseOfCobblestone** [comment]
That's not how this works try again

**Comment by PKPenguin321** [comment]
I can't imagine the national anthem being copyrighted in a significant way

**Comment by fluffe9911** [comment]
Fineeeeeeeee

**Comment by fluffe9911** [comment]
Changed the song into the much more AMERICAN!! U.S.A National Anthem so the copyright gods may have mercy on us but removed the money bleeding cause I dont know how to refactor something into a proc sadly

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 11]
```diff
@@ -1011,6 +1011,49 @@
 		L.apply_status_effect(/datum/status_effect/bonechill)
 		SEND_SIGNAL(L, COMSIG_ADD_MOOD_EVENT, "spooked", /datum/mood_event/spooked)
 
+/datum/species/golem/capitalist
+	name = "Capitalist Golem"
+	id = "capitalist golem"
+	prefix = "Capitalist"
+	attack_verb = "monopoliz"
+	limbs_id = "ca_golem"
+	blacklisted = 1
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]:
```suggestion
	blacklisted = TRUE
```

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 12]
```diff
@@ -1011,6 +1011,49 @@
...
+	if(isgolem(target))
+		return
+	if(target.nutrition >= NUTRITION_LEVEL_FAT)
+		target.set_species(/datum/species/golem/capitalist)
+		return
+	target.adjust_nutrition(40)
+
+/datum/species/golem/capitalist/handle_speech(message, mob/living/carbon/human/H)
+	playsound(H, 'sound/misc/mymoney.ogg', 25, 0)
+	return "Hello I like money!"
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]:
```suggestion
	return "Hello, I like money!"
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]
`mymoney.ogg` needs better cropping

**Comment by fluffe9911** [comment]
@Cyberboss done

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**